internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -1224,70 +873,98 @@ func (mc *mac) startSignalHandler() {
 	}()
 }
 
+func (mc *mac) getPushUrl(stream *api.Stream, targetRef *api.MultistreamTargetRef) (*api.MultistreamTarget, string, error) {
+	target, err := mc.lapi.GetMultistreamTarget(targetRef.ID)
+	if err != nil {
+		return nil, "", fmt.Errorf("error fetching multistream target %s: %w", targetRef.ID, err)
+	}
+	// Find the actual parameters of the profile we're using
+	var videoSelector string
+	// Not actually the source. But the highest quality.
+	if targetRef.Profile == "source" {
+		videoSelector = "maxbps"
```

Comment:
We just hit a bug with this in MistProcLivepeer where sometimes a transcoded rendition can be higher bitrate. @Thulinma is there a correct way for us to select just the source hre?

Reply:
FTR as we chatted about this, this is not a behavior change in this PR, it was only copied from the previous code. We can merge this as is and follow up with a fix for this later.

Reply:
MistProcLivepeer, since an update we rolled out during the holidays, can no longer select its own renditions as source for itself. That _should_ handle this situation without any changes needed.

To answer the actual question: yes, sort-of. You can simply _not_ specify any specific track selector (leave it default value), and set the default track selector to `id_lth` to prioritize the lowest-ID track of the type, which for live streams is guaranteed to be the source (or one of the source tracks, at least, if there is more than one video track in the source already).